app/service/file_svc.py

```diff
@@ -96,6 +114,18 @@ def __init__(self, plugins, exfil_dir):
         """
         self.special_payloads[name] = func
 
+    async def find_payloads(self):
+        """
+        Identify the full gambit of available payloads (filtering out adversary mode)
+        :return: list of available payloads
+        """
+        listing = []
+        for plugin in self.plugins:
+            for root, _, files in os.walk('plugins/%s' % plugin):
+                if root.endswith('payloads') and 'adversary' not in root:
+                    listing.extend(files)
+        return listing
```

Comment:
Clean function.  Also make sure to check the top-level `data/payloads` folder we will create